morpheus/utils/logger.py

```diff
@@ -125,16 +126,23 @@ def _configure_from_log_level(log_level: int):
     # handler
     morpheus_logger.addHandler(morpheus_queue_handler)
 
+    llm_agent_logger = get_llm_agent_logger()
+
+    # This logger will allways log at the debug level, don't propagate upstream
+    llm_agent_logger.propagate = False
+    llm_agent_logger.setLevel(logging.DEBUG)
+
     log_file = os.path.join(appdirs.user_log_dir(appauthor="NVIDIA", appname="morpheus"), "morpheus.log")
+    llm_agent_log_file = os.path.join(appdirs.user_log_dir(appauthor="NVIDIA", appname="morpheus"), "agent.log")
 
     # Ensure the log directory exists
     os.makedirs(os.path.dirname(log_file), exist_ok=True)
 
     # Now we build all of the handlers for the queue listener
-    file_handler = logging.handlers.RotatingFileHandler(filename=log_file, backupCount=5, maxBytes=1000000)
-    file_handler.setLevel(logging.DEBUG)
-    file_handler.setFormatter(
-        logging.Formatter('%(asctime)s - [%(levelname)s]: %(message)s {%(name)s, %(threadName)s}'))
+    file_handler = _get_file_handler(log_file)
+    llm_agent_file_handler = _get_file_handler(llm_agent_log_file)
+    llm_agent_file_handler.addFilter(logging.Filter(name=LLM_AGENT_LOG_NAME))
+    llm_agent_logger.addHandler(llm_agent_file_handler)
```

Comment:
Fyi - This [PR](https://github.com/nv-morpheus/Morpheus/pull/1463) updates logger so that you can pass in additional handlers.